subsys/bluetooth/controller/ll_sw/ull_llcp_pdu.c

```diff
@@ -726,7 +729,7 @@ void llcp_pdu_encode_cte_req(struct proc_ctx *ctx, struct pdu_data *pdu)
 
 void llcp_pdu_decode_cte_rsp(struct proc_ctx *ctx, const struct pdu_data *pdu)
 {
-	if (pdu->cp == 0U || pdu->cte_info.time == 0U) {
```

Comment:
Not sure about the KConfigs - but will octet3.cte_info not be undef'ed for OPENISA
(sorry if I'm missing the obvious!)

Reply:
CTE is not supported in OPENISA, and these lines are conditionally compiled out.

Reply:
That is a very satisfying answer ;-)